ui/src/components/DashboardMetrics.js

```diff
@@ -0,0 +1,234 @@
+import React, { useCallback } from 'react';
+import { useQuery } from 'react-query';
+import { LineChart, Loader } from '@scality/core-ui';
+import { lighten, darken } from 'polished';
+
+import { refreshNodesAction, stopRefreshNodesAction } from '../ducks/app/nodes';
+import { useRefreshEffect, fromUnixTimestampToDate } from '../services/utils';
+import { useTypedSelector } from '../hooks';
+import {
+  queryNodeCPUMetrics,
+  queryNodeMemoryMetrics,
+  queryNodeLoadMetrics,
+} from '../services/prometheus/fetchMetrics';
+import { LAST_TWENTY_FOUR_HOURS, LAST_ONE_HOUR } from '../constants';
+import {
+  yAxisUsage,
+  yAxis,
+  getTooltipConfig,
+} from '../components/LinechartSpec';
+import {
+  GraphTitle,
+  GraphWrapper,
+} from '../components/style/CommonLayoutStyle';
+import { useDynamicChartSize } from '../services/utils';
+
+// import styled from 'styled-components';
+
+const DashboardMetrics = () => {
+  // Making sure nodes list is loaded
+  useRefreshEffect(refreshNodesAction, stopRefreshNodesAction);
+  const nodes = useTypedSelector((state) => state.app.nodes.list);
+  const theme = useTypedSelector((state) => state.config.theme);
```

Comment:
We would eventually get rid of redux.  So I suggest using `react-query` to achieve it.